data/json/npcs/talk_tags.json

```diff
@@ -1122,7 +1492,9 @@
       "Pass some ethanol, I need to power my ethanol burner.",
       "Waiter!  I need a refill, my ethanol burner is running out of charge!",
       "I require ethanol for my internal power supply.  Anything on you?",
-      "Got any alcohol to spare?  Need to recharge my drives.  Methanol, would do."
+      "Got any alcohol to spare?  Need to recharge my drives.  Methanol, would do.",
```

Comment:
`Methanol would do`

Reply:
This was an old line, I think either way it makes sense however.